rskj-core/src/main/java/co/rsk/net/handler/TxValidator.java

```diff
@@ -90,7 +82,7 @@ public TxValidator(RskSystemProperties config, Repository repository, Blockchain
 
             for (TxValidatorStep step : validatorSteps) {
                 if (!step.validate(tx, state, blockGasLimit, minimumGasPrice, bestBlockNumber, basicTxCost == 0)) {
-                    logger.info("Tx validation failed: validator {} tx={}", step.getClass().getName(), tx.getHash());
+                    logger.debug("[tx={}] {} failed", tx.getHash(), step.getClass());
```

Comment:
i think validation fail is a bit more important than debug, i would go for info or warning

Reply:
Agreed. I think `info` would be more suitable given that we probably don't want to give a warning on a triggerable condition, although I'm not sure how we're currently treating warnings, so that counts too. Makes sense?